lib/routes/sms.js

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var logger = require('../logging')('routes.sms');
+var basket = require('../basket');
+
+module.exports = function sms(req, res) {
+  if (! res.locals.creds) {
+    logger.error('auth.missing-authorization-header');
+    res.status(400).json(basket.errorResponse('missing authorization header', basket.errors.USAGE_ERROR));
+    return;
+  }
+
+  var params = req.body;
+  logger.info('params', params);
```

Comment:
IIUC, this will cause us to log some new PII in the form of mobile phone numbers.  Perhaps we should put this down to `debug` level rather than figuring out the details of handling that in production.